lib/pipely/build/daily_scheduler.rb

```diff
@@ -7,7 +7,7 @@ module Build
     class DailyScheduler
 
       def initialize(start_time)
-        @start_time = start_time
+        @start_time = DateTime.parse(start_time).strftime('%H:%M:%S')
```

Comment:
Would it make more sense to compare DateTime vs DateTime instead of String vs String?